src/app/shared/components/cluster-summary/style.scss

```diff
@@ -104,9 +108,16 @@ $steps: 6;
 }
 
 .step {
-  padding-bottom: 24px;
+  &:not(:last-child) {
+    padding-bottom: 48px;
```

Comment:
Just wanted to inform Applications Icon is missing in front of version may be it will be fixed in other PR but discuss with @Waseem826 before adding this change.

![icons-missing](https://user-images.githubusercontent.com/17727069/201315761-7fac3344-b5f0-488b-8f7d-7908c748ee94.png)

Reply:
That's not related to this PR and I'll create separate issue for that.